apps/namadillo/src/App/Governance/AllProposalsTable.tsx

```diff
@@ -299,6 +304,9 @@ const Status: React.FC<CellProps> = ({ proposal }) => (
   <StatusLabel status={proposal.status} className="ml-auto" />
 );
 
-const VotingEnd: React.FC<CellProps> = ({ proposal }) => (
-  <div className="text-right">Epoch {proposal.endEpoch.toString()}</div>
+const VotingEnd: React.FC<CellProps> = ({ proposal: { endTime } }) => (
+  <div className="text-right">
+    <div>{showTime(endTime)}</div>
+    <div className="text-[#878787]">{showDate(endTime)}</div>
```

Comment:
Can we reuse `text-neutral-400` or `text-neutral-450` here?

Reply:
The theme file has `400: "#999999"` and `450: "#838383"` so it looks like we don't have a named colour for `#878787` yet. Do you think I should use 400 or 450 anyway (changing the colour) or define something new? I was getting a bit confused with the designs before because there are a lot of different greys but I wasn't sure if they should all be distinct.

Reply:
No worries, sometimes I get confused too. I think the best approach is to add the closest color we have defined in our theme file. This way we avoid adding so many different shades and make the final UI without standard colors